graphium/config/_loader.py

```diff
@@ -123,6 +123,7 @@ def load_datamodule(
             model_name=config["constants"]["name"],
             gradient_accumulation=config["trainer"]["trainer"].get("accumulate_grad_batches", None),
             ipu_inference_opts=ipu_inference_opts,
+            precision=config["trainer"]["trainer"]["precision"]
```

Comment:
To handle when precision is not given

```suggestion
            precision=config["trainer"]["trainer"].get("precision")
```